aries-site/src/pages/components/button.mdx

```diff
@@ -192,3 +200,74 @@ Button label text size may be specified. Padding and rounding scale proportionat
 >
   <ButtonSizingExample />
 </Example>
+
+## Responsiveness
+
+Ensure you are always being mindful of our responsive page layouts when designing user experiences. Even if you don’t anticipate your users leveraging mobile or tablet devices, they may be reducing/expanding browser widths, zooming in on content, or manipulating the browser in unanticipated ways. As a best practice, always design for a usable UI on all browser widths and resolutions.
+
+### Best Practices
+- Ensure the button label does not wrap, truncate, or alter the text content in any way when presented on smaller browser widths.
+- Button label text should not scale down to fit on smaller browser widths.
+- The minimum button width on mobile should be 42px.
+
+### Call to Action
+
+When buttons are “Brand” or “Call to Action (CTA)”, it is best to have them fill the full width of the container for browser widths of "small". Because press target size and an unclear visual hierarchy are key concerns here, this utilizes available space to ensure users can easily identify and interact with important buttons.
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{type: 'do', message: 'This secondary button is not a CTA more subtly presents itself on the page.'}}
+  >
+    <ButtonGoodCTAPreview />
+  </Example>
+  <Example
+    bestPractice={{type: 'dont', message: 'This secondary button draws unnecessary attention to itself and confuses the visual heirarchy of the page.'}}
+  >
+    <ButtonBadCTAPreview />
+  </Example>
+</BestPracticeGroup>
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{type: 'do', message: 'Full width CTAs help users identify & interact with buttons easier on smaller browser widths and when using the press gesture.'}}
```

Comment:
Is this on desktop also, or just "small" screen? If so, we might want to call it out better like "On 'small' browser widths, full width CTAs..." otherwise it could get confusing for folks that might just scan visuals.